<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Bharath" failures="1" tests="5" name="Test" time="0.055" errors="0" timestamp="2022-12-08T21:15:38 IST">
  <testcase classname="framework.TestNgTestClass" name="testcase4" time="0.003">
    <skipped/>
  </testcase> <!-- testcase4 -->
  <testcase classname="framework.TestNgTestClass" name="testcase4" time="0.001">
    <skipped/>
  </testcase> <!-- testcase4 -->
  <testcase classname="framework.TestNgTestClass" name="testcase4" time="0.001">
    <failure type="java.lang.Exception" message="Failed">
      <![CDATA[java.lang.Exception: Failed
at framework.TestNgTestClass.testcase4(TestNgTestClass.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="framework.TestNgTestClass" name="testcase1" time="0.001"/>
  <testcase classname="framework.TestNgTestClass" name="testcase2" time="0.002"/>
  <testcase classname="framework.TestNgTestClass" name="testcase3" time="0.0">
    <ignored/>
  </testcase> <!-- testcase3 -->
</testsuite> <!-- Test -->
